svg/elements/feComposite.json

```diff
@@ -24,16 +24,16 @@
               "version_added": null
             },
             "opera": {
-              "version_added": false
+              "version_added": true
```

Comment:
Note, these must all be true/supported because you can't have support for "lighter" if the parent item is not supported (as per bugs). I am not sure what versions. 

If you believe chrome then this was in from release 1 (webkit and from then in Blink). Should I leave these alone or assume that support came in earliest matching version of Blink/Webkit for the other browsers? That would mean, for example, that Opera would change to version 15 and I'd assume there was no support in Presto. Thoughts?